### ISSUE TITLE
How to parse ProXL data with CrossLinkDataBase?

### ISSUE THREAD

**Issue by cgreenberg** [salilab/pmi#176]
@Pellarin I don't know how to parse these files with your DB object, can you advise? They're tab-delimited and the term "POSITION" appears twice in the header. This is the format from the popular Xl viewer site ProXL. Advice?

```SEARCH ID(S) TYPE    PROTEIN 1   POSITION    PROTEIN 2   POSITION    NUM PSMS    NUM PEPTIDES    NUM UNIQUE PEPTIDES BEST PEPTIDE Q-VALUE    BEST PSM Q-VALUE
240 CROSSLINK   Spc97   24  Spc97   39  13  1   1   0.003785    3.514E-4

```
```

**Comment by Pellarin** [comment]
can you send me a file?

**Comment by Pellarin** [comment]
is it a fixed format?

**Comment by Pellarin** [comment]
closed by f782613e45c66c48c10c9ca4f6165d35013049d0

**Comment by Pellarin** [comment]
see test/test_io_crosslink.py  the function: test_xlpro_fixed_format_delimited
